CIP-ledger-evolution/README.md

```diff
@@ -0,0 +1,302 @@
+
+#### Ledger protocol
+
+The `ledger-protocol` category corresponds closely to block header validation.
+It corresponds precisely to all the functionality in the Shelley ledger specification
+not captured by the `BBODY` and `TICK` transitions.
+This mostly involves handling the nonces for Ouroboros, the VRF checks,
+the KES certificates, and block leadership checks.
+
+Historically, the ledger protocols have been specified alongside the ledger eras.
+The `TPraos` ledger protocol was specified alongside Shelley, and
+`Praos` was specified alongside Babbage.
+
+To become active, a ledger protocol change MUST be
+specified with the STS framework and MUST be included in the software update for a soft/hard fork.
+If the fork coincides with a ledger era, the specifications can be combined.
+
+#### Ledger eras
+
+A `ledger-era` is a collection of ledger-features, bug fixes, intra-era switches, and other minor changes
+grouped together with a name and activated together at a fork.
+In the event of interaction between the features, details may need to be provided.
+
+To become active, the ledger era MUST:
+* specify the semantics with the [STS](STS.md) framework
+* specify cryptographic details in an appendix to the STS specification
+* specify the CBOR serialization schema with a CDDL file
+* be included in the software update for a soft/hard fork.
+
+### Serialization changes
+
+See [Serialization](Serialization.md).
+
+### The Small Step Semantics Framework for Cardano
+
+See [STS](STS.md).
+
+### Implementing and releasing changes
+
+This CIP does not cover the process of implementing changes.
+As usual, the CIP process covers the design phase, and it is up to the implementer to ensure that
+their proposal is implemented, which may require additional work to meet the requirements of the
+maintainers of the Cardano code repositories (testing, implementation quality, approach), and so on.
+
+Changes can be released after their CIPs have reached Active status.
+Different changes will require different releases as described in "Types of release".
+This CIP does not cover the process by which changes are actually incorporated into releases after having been implemented.
+In particular, there is NO assumption that a feature which requires a particular release will be
+included in the next such release, even after it has been implemented.
+
+### Ledger Core CIP registry
+
+Any CIP which proposes a type of change listed in "Types of change" MUST also add itself to this registry (in addition to the main registry).
+
+| #  | Title             | Type of change          | Status |
+|----|-------------------|-------------------------|--------|
+| 31 | Reference inputs  | Ledger-script interface | Draft  |
+| 32 | Inline datums     | Ledger-script interface | Draft  |
+| 33 | Reference scripts | Ledger-script interface | Draft  |
+
+## Rationale
+
+### Why have a public process for changes?
```

Comment:
this isn't necessary in the light of the new CIP-001, I think

Reply:
I thought it was. from CIP-1:

> Additionally, projects of the ecosystem may explicitly enlist as new categories. The following section describes how projects can engage with the CIP process.

Reply:
I mean, you don't need to _justify_ why you're doing it, CIP-001 has the justificatoin!

Reply:
gotcha! I think I stole that from CIP-35 :grin: